src/main/java/randoop/main/GenTests.java

```diff
@@ -788,33 +800,10 @@ private void logOutputStart(int sequenceCount, String testKind) {
   private void logTestFiles(List<File> files) {
     System.out.println();
     for (File f : files) {
-      System.out.println("Created file: " + f.getAbsolutePath());
-    }
-  }
-
-  /**
-   * Writes the test driver for the test classes created by the JUnit creator.
-   *
-   * @param junitPrefix the prefix of the driver class name
-   * @param junitCreator the {@link JUnitCreator} used to create test classes
-   * @param testClassNames the set of names of the generated test classes
-   * @param jfw the writer to output the Java file for the test driver
-   * @return the {@code File} for the test driver Java file
-   */
-  private File outputTestDriver(
-      String junitPrefix,
-      JUnitCreator junitCreator,
-      Set<String> testClassNames,
-      JavaFileWriter jfw) {
-    String classSource;
-    String driverName = junitPrefix;
-    if (GenInputsAbstract.junit_reflection_allowed) {
-      classSource = junitCreator.createSuiteClass(driverName, testClassNames);
-    } else {
-      driverName = junitPrefix + "Driver";
-      classSource = junitCreator.createTestDriver(driverName, testClassNames);
+      if (f != null) {
```

Comment:
For my edification, why is the method responsible for documenting the behavior of a client?